src/main/resources/org/jenkinsci/plugins/displayurlapi/user/PreferredProviderUserProperty/config.jelly

```diff
@@ -2,7 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:block>
-    <f:entry field="providerId">
+    <f:entry field="providerId" title="${%Notification URL}">
```

Comment:
not 100% ideal as it repeats the section title, but better than what's there currently

<img width="999" alt="image" src="https://github.com/jenkinsci/display-url-api-plugin/assets/21194782/3ff1355c-c92f-4ad3-a85c-92dacfa25925">